build-bin/maven/maven_release

```diff
@@ -25,19 +25,10 @@ export MAVEN_OPTS="$($(dirname "$0")/maven_opts)"
 
 trigger_tag=${1?trigger_tag is required. Ex release-1.2.3}
 release_version=$(build-bin/git/version_from_trigger_tag release- ${trigger_tag})
-release_branch=${2:-master}
```

Comment:
Once we lost the branch information, we can't bump up the latest of codes. The script may exit with error status.

Reply:
I have modified the branch selection policy here. It will now only extract the branch where the last commit within the current tag.